ItemDrop/data/tables/itemdrop2-sct.tbm

```diff
@@ -336,9 +336,10 @@ function Items:DeathDropMaybe(ship)
 		return
 	end
 
-	if ship:getBreedName() ~= "Ship" then
-		return
-	end
+	--No longer needed. We're guaranteed by the hook that this is a ship
+	--if ship:getBreedName() ~= "Ship" then
+	--	return
+	--end
```

Comment:
Personally I'd prefer to delete the section entirely rather than accrue dead code comments in this case.